src/structstore_shared.hpp

```diff
@@ -44,116 +50,138 @@ class StructStoreShared {
         ~SharedData() = delete;
     };
 
-    std::string shm_path;
-    int shm_fd;
-    SharedData* shm_ptr;
-    bool owning;
+    std::string path;
+    int fd;
+    SharedData* ptr;
```

Comment:
This name is now very generic, maybe something like `sh_data_ptr` is clearer? But I'm also fine with leaving it at `ptr`

Reply:
Agreed, will be changed!